docs/releases.rst

```diff
@@ -3,6 +3,133 @@
+In addition, this is the first version to work correctly with PostgreSQL 10.
+
+**Custom bootstrap**
+
+- Make the process of bootstrapping the cluster configurable (Alexander Kukushkin)
+
+  Allow custom bootstrap scripts instead of ``initdb`` when initializing the very first node in the cluster.
+  The bootstrap command receives the name of the cluster and the path to the data directory. The resulting cluster can
+  be configured to start in recovery, making it possible to bootstrap a new cluster from the point in the timeline
+  of an existing one. Refer to the :ref:`documentaton page<custom_bootstap>` for more detailed description of this feature.
+
+**Smarter pg_rewind support**
+
+-  Decide on whether to run pg_rewind by looking at the timeline differences from the current master (Alexander)
+
+   Previously, Patroni had a fixed set of conditions to trigger pg_rewind, namely when starting a former master, when
+   doing a switchover to the designated node that is not a master or when there is a replica with the nofailover tag,
+   basically those cases when there is a chance that a replica may be ahead of the new master. In some cases, pg_rewind
+   did nothing, in some other ones it was not running when necessary. Instead of relying on this limited list of rules
+   this change makes Patroni compare the master and the replica WAL positions (using the streaming replication protocol)
+   in order to reliably decide if the replica to join is ahead of the new master.
+
+**Synchronous replication mode strict**
+
+-  Enhance synchronous replication support by adding the strict mode (James Sewell, Alexander)
+
+   When the new configuration option ``synchronous_mode_strict`` is set, Patroni will not let the master accept write
+   requests if there are no replicas attached . This option provides the guarantee that no writes will be lost because
+   of the failover. When a new replica attaches to the cluster running in the strict synchronous mode, Patroni changes its
+   synchronous standby names to point to that replica and shortly afterwards sets it as a synchronous replica in DCS.
+   If the master dies in-between, there will be a synchronous replica retaining the writes, but Patroni will not be able
+   to pick that replica for the failover. In that case, one should perform a manual promotion to such a replica.
+
+**Configuration editing with patronictl**
+
+- Add configruation editing to patronictl (Ants Aasma, Alexander)
+
+  Add the ability to patronictl of editing dynamic cluster configuration stored in DCS. Support either specifying the
+  parameter/values from the command-line, involing the $EDITOR, or applying configuration from the yaml file.
+
+**Linux watchdog support**
+
+- Implement wachdog support for Linux (Ants)
+
+  Support Linux software watchdog in order to reboot the master node that losses the lock in Etcd. Normally, Patroni will
```

Comment:
> reboot the master node that losses the lock in Etcd.

Watchdog doesn't know anything about Etcd, the main purpose of watchdog is tor reboot the node is Patroni is not running or hanging.